frontend/src/routes/Infrastructure/InfraEnvironments/CreateInfraEnv.tsx

```diff
@@ -6,17 +6,15 @@ import { useState } from 'react'
 import { useTranslation } from '../../../lib/acm-i18next'
 import MonacoEditor from 'react-monaco-editor'
 import { useHistory } from 'react-router'
-import TemplateEditor from 'temptifly'
-
+import TemplateEditor from '../../../components/TemplateEditor'
 import { NavigationPath } from '../../../NavigationPath'
 import infraEnvTemplate from './infraenv-template.hbs'
 import InfraEnvForm, { Portals } from './InfraEnvForm'
 
 // include monaco editor
 import 'monaco-editor/esm/vs/editor/editor.all.js'
 import 'monaco-editor/esm/vs/basic-languages/yaml/yaml.contribution.js'
-import '../Clusters/ManagedClusters/CreateCluster/style.css'
-import 'temptifly/dist/styles.css'
+import '../../Applications/CreateApplication/Subscription/style.css'
```

Comment:
What is the reason for this change? The previously reference stylesheet is still used elsewhere.

Reply:
that style.css is used to position the template editor --- but the one in createcluster is good for when it's a wizard while the one in app subscription is good for when it's a form--and infra is using it as a form